var/spack/repos/builtin/packages/rocm-hip-clang/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class RocmHipClang(CMakePackage):
+    """HIP-Clang: llvm and clang for ROCm"""
+
+    homepage = "https://github.com/RadeonOpenCompute/llvm-project"
+    url = "https://github.com/RadeonOpenCompute/llvm-project/archive/rocm-3.5.0.tar.gz"
+
+    version('3.5.0', '4878fa85473b24d88edcc89938441edc85d2e8a785e567b7bd7ce274ecc2fd9c')
+    depends_on('gcc')
```

Comment:
I'm not sure how I feel about this. Compilers aren't really modeled as dependencies just yet. Does it only work with GCC?

Reply:
This is my lame attempt to deal with issues surrounding lib(std)c++ and the fact that clang/llvm happily grabs onto the system GCC if not appropriately guided. (I have been looking for something more elegant in this space as well)

Reply:
Put another way, if we mix libc++ and libstdc++ here we have issues. The "solution"/intent here was to force everything in the chain to be libstdc++.